packages/wordpress-playground-block/src/i18n.ts

```diff
@@ -0,0 +1,9 @@
+import { createI18n, sprintf as coreSprintf } from '@wordpress/i18n';
+
+const i18n = createI18n(undefined, 'interactive-code-block');
```

Comment:
Nice, I didn't know that you could set the domain in one place instead of writing it on every line.

Reply:
> Nice, I didn't know that you could set the domain in one place instead of writing it on every line.

I didn't at first either. This PR started with adding on every line (or discovering that I'd forgotten to add it in a bunch of places). This is much better. ☺️